packages/http/src/validator/utils/__tests__/spec.spec.ts

```diff
@@ -1,16 +1,60 @@
 import { findOperationResponse } from '../spec';
 
 describe('findOperationResponse()', () => {
-  it('works', () => {
-    expect(
-      findOperationResponse(
-        [
-          { code: '2XX', contents: [], headers: [] },
-          { code: '20X', contents: [], headers: [] },
-          { code: '1XX', contents: [], headers: [] },
-        ],
-        200,
-      ),
-    ).toEqual({ code: '20X', contents: [], headers: [] });
+  describe('when response for given code exists', () => {
+    it('returns found response', () => {
+      expect(
+        findOperationResponse(
+          [
+            { code: '2XX', contents: [], headers: [] },
+            { code: '20X', contents: [], headers: [] },
+            { code: 'default', contents: [], headers: [] },
+            { code: '1XX', contents: [], headers: [] },
+          ],
+          200,
+        ),
+      ).toEqual({ code: '20X', contents: [], headers: [] });
```

Comment:
I'm assuming this has been specified because it's the most specific, correct?

Reply:
Yes